cookbook/design/orientation.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: "Updating the UI based on orientation"
+permalink: /cookbook/design/orientation/
+---
+
+In certain cases, it can be handy to update the design of an app when the user
```

Comment:
you could just use OrientationBuilder for this

Reply:
Sounds good. I wasn't sure if that was always the right tool for the job, but happy to go that route if it's more idiomatic.

Reply:
Fixed.